lib/aussieaddonscommon/utils.py

```diff
@@ -211,20 +211,26 @@ def get_kodi_build():
     try:
         return xbmc.getInfoLabel("System.BuildVersion")
     except Exception:
-        return
+        return ''
 
 
 def get_kodi_version():
     """Return the version number of Kodi"""
     build = get_kodi_build()
-    version = build.split(' ')[0]
-    return version
+    if build:
+        version = build.split(' ')[0]
+        return version
+    else:
+        return '0'
 
 
 def get_kodi_major_version():
     """Return the major version number of Kodi"""
     version = get_kodi_version().split('.')[0]
-    return int(version)
+    if version:
```

Comment:
Looks like version should never evaluate to `False` from `get_kodi_version()`, so this change might not be that useful?

Reply:
Yeah I worked from this function upwards, should have come back to look at it again!